src/composables/useCountdown.ts

```diff
@@ -0,0 +1,65 @@
+// libraries
+import { ref, watchEffect, onBeforeUnmount, Ref } from 'vue';
+
+// types
+import { Countdown } from 'src/components/types';
+
+export const useCountdown = (releaseDate: string): { countdown: Ref<Countdown> } => {
+  const countdown = ref<Countdown>({
+    days: 0,
+    hours: 0,
+    minutes: 0,
+    seconds: 0,
+  });
+
+  let countdownInterval: ReturnType<typeof setInterval> | null = null;
+
+  const startCountdown = () => {
+    const targetDate = new Date(releaseDate).getTime();
+
+    countdownInterval = setInterval(() => {
```

Comment:
Missing func return type `setInterval((): void => {`

Reply:
Added return type `void`.
The `countdownInterval` variable is not defined but assigned here - the type declaration is above: [originally line 15](https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/66/files/5b686ab11e008c35f22c519e754e065c8db9e936#diff-e389e87812ecbd40200cb8d8cee4e6d5cf56a66ca84ac316d7c1c5a1e551a72dR15)
I changed its type to `NodeJS.Timeout` for brevity.